### DISCUSSION TITLE
Add ENV flag for disabling File Uploads/RAG

### DISCUSSION THREAD

**Discussion by gaby** [open-webui/open-webui#5722] [General]
**Is your feature request related to a problem? Please describe.**
Currently Open-WebUI allows users to upload files to a chat. This requires using RAG and also downloads a model in the background, and no limits to size.

When running Open-WebUI with multiple users there's a chance someone uploads a massive file affecting users or the host running Open-WebUI. For our use case we do not use the RAG feature and the uploads just sit spinning since the model required is not available.

**Describe the solution you'd like**
Add an ENV variable that allows disabling any kind of file uploads within a Chat.

**Describe alternatives you've considered**
Running a cronjob and deleting any files found in `/data/docs`.

**Additional context**

**Comment by rgaricano** [comment]
You can config it on:
 Admin Panel / Settings / Documents / Files / Max Upload Size & Max Upload Count

(setting to 0 doesn`t permit uploads)

**Reply by gaby** [reply]
Thanks! That should do it.
